modules/ReactBreakpoints.js

```diff
@@ -130,12 +130,12 @@ class ReactBreakpoints extends React.Component {
   componentWillUnmount() {
     if (typeof window !== 'undefined') {
       if (this.props.debounceResize) {
-        window.addEventListener(
+        window.removeEventListener(
           'resize',
           debounce(this.readWidth, this.props.debounceDelay),
```

Comment:
Actually, #35 is better as `debounce()` will return a new function at every call. `removeEventListener()` should be called with the same function to properly remove it.